src/components/NcDateTimePickerNative/NcDateTimePickerNative.vue

```diff
@@ -0,0 +1,320 @@
+			}
+		},
+	}
+</script>
+```
+
+#### Usage: type='month'
+```vue
+<template>
+	<span>
+		<NcDateTimePickerNative
+			v-model="value"
+			:id="id"
+			:label="label"
+			type="month" />
+	</span>
+</template>
+
+<script>
+	export default {
+		data() {
+			return {
+				value: new Date(),
+				id: 'date-time-picker_id',
+				label: 'please select a new date',
+			}
+		},
+	}
+</script>
+```
+
+</docs>
+
+<template>
+	<div class="native-datetime-picker">
+		<label :class="{ 'hidden-visually': hideLabel }" :for="`id_${id}`">{{ label }}</label>
+		<input :id="`id_${id}`"
+			class="native-datetime-picker--input"
+			:type="type"
+			:value="formattedValue"
+			:min="formattedMin"
+			:max="formattedMax"
+			v-on="listeners">
+	</div>
+</template>
+
+<script>
+
+const inputDateTypes = ['date', 'datetime-local', 'month', 'time', 'week']
+
+export default {
+	name: 'NcDateTimePickerNative',
+
+	props: {
+		/**
+		 * value attribute of the input field
+		 * required
+		 */
+		value: {
+			type: [Date, String],
+			required: true,
+		},
+		/**
+		 * id attribute of the input field
+		 * required
+		 */
+		id: {
+			type: String,
+			required: true,
+		},
+		/**
+		 * type attribute of the input field
+		 * default type: date
+		 * The type of the input element, it can be `date`, `datetime-local`, `month`, `time`, `week`
+		 */
+		type: {
+			type: String,
+			default: 'date',
+			validate: (name) => inputDateTypes.includes(name),
+		},
+		/**
+		 * text inside the label element
+		 */
+		label: {
+			type: String,
+			default: 'Please choose a date',
+		},
+		/**
+		 * min attribute of the input field
+		 */
+		min: {
+			type: Date,
+			default: null,
+		},
+		/**
+		 * max attribute of the input field
+		 */
+		max: {
+			type: Date,
+			default: null,
+		},
+		/**
+		 * Flag to hide the label
+		 * The hidden input label for accessibility purposes.
+		 */
+		hideLabel: {
+			type: Boolean,
+			default: false,
+		},
+	},
+
+	computed: {
+		formattedValue() {
+			return this.formatValue(this.value)
+		},
+		formattedMin() {
+			if (this.min) {
+				return this.formatValue(this.min)
+			}
+			return false
+		},
+		formattedMax() {
+			if (this.max) {
+				return this.formatValue(this.max)
+			}
+			return false
+		},
+		listeners() {
+			return {
+				...this.$listeners,
+				/**
+				 * @param $event
+				 * @return new chosen Date()
+				 */
+				input: ($event) => {
+					if (isNaN($event.target.valueAsNumber)) {
+						return this.$emit('input', '')
+					} else {
+						if (this.type === 'time') {
+							const time = $event.target.value
+							if (this.value === '') {
+								// this case is because of Chrome bug
+								const { yyyy, MM, dd } = this.getReadableDate(new Date())
+								return this.$emit('input', new Date(`${yyyy}-${MM}-${dd}T${time}`))
+							} else {
+								const { yyyy, MM, dd } = this.getReadableDate(this.value)
+								return this.$emit('input', new Date(`${yyyy}-${MM}-${dd}T${time}`))
+							}
```

Comment:
```suggestion
							if (this.value === '') {
								// this case is because of Chrome bug
								const { yyyy, MM, dd } = this.getReadableDate(new Date())
								return this.$emit('input', new Date(`${yyyy}-${MM}-${dd}T${time}`))
							}
								
							const { yyyy, MM, dd } = this.getReadableDate(this.value)
							return this.$emit('input', new Date(`${yyyy}-${MM}-${dd}T${time}`))
```